x-pack/plugins/infra/public/containers/with_kuery_autocompletion.tsx

```diff
@@ -85,7 +86,7 @@ class WithKueryAutocompletionComponent extends React.Component<
         query: expression,
         selectionStart: cursorPosition,
         selectionEnd: cursorPosition,
-        indexPatterns: [indexPattern],
+        indexPatterns: [indexPattern as DataView],
```

Comment:
hm... could you please double check that. I see here indexPattern has DataViewBase type. Not sure that it's safe to cast that type